packages/base/src/signals/signal-manager.ts

```diff
@@ -43,6 +44,9 @@ export const Signals = {
 };
 
 export class SignalManager extends EventEmitter implements SignalManager {
+    fireTabChangedSignal(tabName: string, expirementUUID: string): void {
+        this.emit(Signals.TABNAME_CHANGED, {tabName, expirementUUID});
```

Comment:
Should be `{ tabName, experimentUUID });` (typo + space separating the curly braces) and the typo should be corrected in the argument